frontend/src/components/events/sponsors/index.jsx

```diff
@@ -39,18 +14,49 @@ const sponsorsHeading = (
 );
 
 class Sponsors extends Component {
+  componentDidMount() {
+    this.props.gSponsors();
+  }
+
   render() {
     return (
       <div className="sponsor-gallery">
         {sponsorsHeading}
         <Container>
-          {urls.map((url, i) => (
-            <img key={i} src={url} className="logo-picture" alt="sponsor logo" />
-          ))}
+          {this.props.sponsors ? (
```

Comment:
i dont think we need the `loading sponsors` section because of our `initialState`, we can just have this be 

```this.props.sponsors && ... ```